### PR TITLE
Remove console-api CRUD queries from search pages

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Related Issue: https://github.com/open-cluster-management/backlog/issues/17364

### CODE REVIEW

File: frontend/src/resources/managedclusteraction.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                        result: actionResponse.status?.result,
+                    })
+                } else if (isComplete === 'Completed' && isActionDone !== 'ActionDone') {
+                    reject({ message: actionMessage })
+                }
+                deleteManagedClusterAction({ name: actionName, namespace: clusterName })
+            })
+            .catch((err) => {
+                if (retries-- > 0) {
+                    setTimeout(poll, 500, resolve, reject)
```
Comment (KevinFCormier) [inline frontend/src/resources/managedclusteraction.tsx:162]: Should we make the poll interval configurable? In console-api, looks like the default was 100, not 500.

Comment (zlayne) [reply frontend/src/resources/managedclusteraction.tsx:162]: I can update it to 100. In looking at the console-api deployment we are never overriding the default value.

**Pull request merged**